target/linux/ramips/dts/mt7628an_asus_rtn600.dts

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rtn600", "mediatek,mt7628dan";
+	model = "ASUS RT-N600";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};
+
+	leds {
```

Comment:
possibly missing some leds? there're 9 leds in total according to user's manual:
https://dlcdnets.asus.com/pub/ASUS/wireless/RT-N600/E11273_RT_N600_Manual.pdf